TACC/tup-ui
# feat(tup-ui): login, Security and Policies links

## Overview

- Make "Security" link work.
- Disable policies link until further notice.

## Related

None

## Changes

- add URL to "Security" page
- disable link to "Policies" page/feature

## Testing

1. Verify login page "Security" link URL would load https://dev.tup.tacc.utexas.edu/about/security-and-compliance/protected-data-service/.
2. Verify "Policies" link is disabled.

## UI

![login footer links](https://user-images.githubusercontent.com/62723358/219213817-572a7cb9-e304-4b41-9dba-4ba5e989bbfc.png)
